devel/cppcheck/Portfile

```diff
@@ -51,76 +52,16 @@ notes                       "If you've updated cppcheck from a previous version,
-    copy ${worksrcpath}/cfg/motif.cfg ${destroot}${prefix}/share/cppcheck/motif.cfg.dist
-    copy ${worksrcpath}/cfg/posix.cfg ${destroot}${prefix}/share/cppcheck/posix.cfg.dist
-    copy ${worksrcpath}/cfg/qt.cfg ${destroot}${prefix}/share/cppcheck/qt.cfg.dist
-    copy ${worksrcpath}/cfg/sdl.cfg ${destroot}${prefix}/share/cppcheck/sdl.cfg.dist
-    copy ${worksrcpath}/cfg/sfml.cfg ${destroot}${prefix}/share/cppcheck/sfml.cfg.dist
-    copy ${worksrcpath}/cfg/std.cfg ${destroot}${prefix}/share/cppcheck/std.cfg.dist
-    copy ${worksrcpath}/cfg/windows.cfg ${destroot}${prefix}/share/cppcheck/windows.cfg.dist
-    copy ${worksrcpath}/cfg/wxwidgets.cfg ${destroot}${prefix}/share/cppcheck/wxwidgets.cfg.dist
-}
-
-post-activate {
-    if {![file exists ${prefix}/share/cppcheck/avr.cfg]} {
-        copy ${prefix}/share/cppcheck/avr.cfg.dist ${prefix}/share/cppcheck/avr.cfg
-    }
-
-    if {![file exists ${prefix}/share/cppcheck/bsd.cfg]} {
-        copy ${prefix}/share/cppcheck/bsd.cfg.dist ${prefix}/share/cppcheck/bsd.cfg
-    }
-
-    if {![file exists ${prefix}/share/cppcheck/cppcheck-cfg.rng]} {
-        copy ${prefix}/share/cppcheck/cppcheck-cfg.rng.dist ${prefix}/share/cppcheck/cppcheck-cfg.rng
-    }
-
-    if {![file exists ${prefix}/share/cppcheck/gnu.cfg]} {
-        copy ${prefix}/share/cppcheck/gnu.cfg.dist ${prefix}/share/cppcheck/gnu.cfg
-    }
-
-    if {![file exists ${prefix}/share/cppcheck/gtk.cfg]} {
-        copy ${prefix}/share/cppcheck/gtk.cfg.dist ${prefix}/share/cppcheck/gtk.cfg
-    }
-
-    if {![file exists ${prefix}/share/cppcheck/microsoft_sal.cfg]} {
-        copy ${prefix}/share/cppcheck/microsoft_sal.cfg.dist ${prefix}/share/cppcheck/microsoft_sal.cfg
-    }
 
-    if {![file exists ${prefix}/share/cppcheck/motif.cfg]} {
-        copy ${prefix}/share/cppcheck/motif.cfg.dist ${prefix}/share/cppcheck/motif.cfg
-    }
-
-    if {![file exists ${prefix}/share/cppcheck/posix.cfg]} {
-        copy ${prefix}/share/cppcheck/posix.cfg.dist ${prefix}/share/cppcheck/posix.cfg
-    }
-
-    if {![file exists ${prefix}/share/cppcheck/qt.cfg]} {
-        copy ${prefix}/share/cppcheck/qt.cfg.dist ${prefix}/share/cppcheck/qt.cfg
-    }
-
-    if {![file exists ${prefix}/share/cppcheck/sdl.cfg]} {
-        copy ${prefix}/share/cppcheck/sdl.cfg.dist ${prefix}/share/cppcheck/sdl.cfg
-    }
-
-    if {![file exists ${prefix}/share/cppcheck/sfml.cfg]} {
-        copy ${prefix}/share/cppcheck/sfml.cfg.dist ${prefix}/share/cppcheck/sfml.cfg
-    }
-
-    if {![file exists ${prefix}/share/cppcheck/std.cfg]} {
-        copy ${prefix}/share/cppcheck/std.cfg.dist ${prefix}/share/cppcheck/std.cfg
-    }
-
-    if {![file exists ${prefix}/share/cppcheck/windows.cfg]} {
-        copy ${prefix}/share/cppcheck/windows.cfg.dist ${prefix}/share/cppcheck/windows.cfg
+    foreach f [glob -directory ${worksrcpath}/cfg *] {
+        copy $f ${destroot}${prefix}/share/cppcheck/[file tail $f].dist
```

Comment:
Use `[glob -tails -directory ${worksrcpath}/cfg *]` and `copy ${worksrcpath}/cfg/${f} .../${f}.dist` can save several `file tail` calls.